server/test/artillery.yml

```diff
@@ -3,13 +3,21 @@ config:
   phases:
     - duration: 20
       arrivalRate: 20
-  defaults:
-    headers:
-      x-my-service-auth: '987401838271002188298567'
+  # defaults:
+  #   headers:
+      # x-my-service-auth: '987401838271002188298567'
 scenarios:
   - flow:
-    - get:
-        url: "/locations/1/reviews"
+    - loop:
+      - get:
+          url: "/locations/ {{ $loopElement }}/reviews"
+      over: 
+        - 9000000
+        - 9500000
+        - 9999999
```

Comment:
Does this loop over those 3 numbers or does it pick randomly from within those 3 ranges? I think you'd want to randomly hit different ids with a separate csv, if you have time.